From 51246818af1f41b3b48fae8cb35be5c1bec69799 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Thu, 8 Feb 2018 19:43:30 +0800
Subject: [PATCH] arm64: dts: rockchip: remove HCLK_HOST_ARB from usb host for
 px30

The HCLK_HOST_ARB is enabled with CLK_IGNORE_UNUSED flag,
so we don't need to control this clk in usb host EHCI and
OHCI controller driver.

Change-Id: I25201170977b30f904611bbcfdcb1eb6230195fe
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index dcf3959bb97c..ca4420cf2dbd 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -859,8 +859,7 @@
 		compatible = "generic-ehci";
 		reg = <0x0 0xff340000 0x0 0x10000>;
 		interrupts = <GIC_SPI 60 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&cru HCLK_HOST>, <&cru HCLK_HOST_ARB>,
-			 <&u2phy>;
+		clocks = <&cru HCLK_HOST>, <&u2phy>;
 		clock-names = "usbhost", "arbiter", "utmi";
 		power-domains = <&power PX30_PD_USB>;
 		phys = <&u2phy_host>;
@@ -872,8 +871,7 @@
 		compatible = "generic-ohci";
 		reg = <0x0 0xff350000 0x0 0x10000>;
 		interrupts = <GIC_SPI 61 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&cru HCLK_HOST>, <&cru HCLK_HOST_ARB>,
-			 <&u2phy>;
+		clocks = <&cru HCLK_HOST>, <&u2phy>;
 		clock-names = "usbhost", "arbiter", "utmi";
 		power-domains = <&power PX30_PD_USB>;
 		phys = <&u2phy_host>;
-- 
2.35.3

